packages/notifi-react-card/lib/components/subscription/subscription-card-views/HistoryCardView.tsx

```diff
@@ -34,28 +35,48 @@ export type AlertHistoryViewProps = Readonly<{
     notificationList?: string;
     emptyAlertsBellIcon?: string;
   }>;
+  icons?: Readonly<{
+    healthCheckIcon?: JSX.Element;
+    broadcastIcon?: JSX.Element;
+  }>;
```

Comment:
i don't feel that this is too useful exposed as Props -- let's remove configurability until someone asks for it.

Reply:
removed icon props